routes/index.js

```diff
@@ -21,6 +22,78 @@ router.get("/type", function(req, res, next) {
   res.render("type");
 });
 
+router.get(/^\/(admin|dashboard)\/bug$/, function(req, res, next) {
```

Comment:
How about extracting the regex in a separate constant `RegExp` object and using the constant throughout? This will increase the readability of the code, making changes (if required) in the future will be easier.